### DISCUSSION TITLE
Are you looking forward to Avalonia supporting the OpenHarmony platform?

### DISCUSSION THREAD

**Discussion by chatterzhao** [AvaloniaUI/Avalonia#17060] [Polls]
> Statistics released by Counterpoint on September 9 show that in the first quarter of 2024, [OpenHarmony](https://gitee.com/openharmony)'s operating system accounted for 4 percent of the global market and surpassed iOS as the second largest operating system in the Chinese market.The increase in market share is attributed to the strong sales of Huawei's flagship products.(Wall Street)
>
> "The number of  OpenHarmony ecosystem devices has exceeded 900 million as of June 2024."

Reference：
- [ohos_flutter](https://gitee.com/openharmony-sig/flutter_flutter)
- [ohos_React_Native](https://gitee.com/openharmony-sig/ohos_react_native/blob/0.72.5-ohos-5.0-release/docs/架构介绍.md)
- [ArkUI-X](https://gitee.com/arkui-x)
- [ohos_QT](https://gitee.com/openharmony-sig/qt)
- [ohos_Avalonia](https://github.com/CeSun/OpenHarmony.Avalonia)

**Comment by rabbitism** [comment]
Please let me know if you are in touch with anyone from OH that is responsible for this matter.

**Comment by MikeCodesDotNET** [comment]
It's news to me (and the team) that we will be supporting this....

**Comment by thevortexcloud** [comment]
For previous discussions see: #16377 and #14019

**Comment by kyuranger** [comment]
I am not optimistic about this, mainly for several reasons:
1. I think Avalonia will not provide official support (Avalonia officials are very busy and have no time to do more). Now there are different people in the community doing it, but they are all independent projects fighting on their own, without a unified project, and actually doing a lot of repetitive work and very inefficient.
2. Because of some other commercial projects, I have contacted HUAWEI engineers. Their attitude towards me is quite arrogant, and the .NET technology stack has always been despised in China (most of the technology stacks used in China are front-end, FLUTTER, JAVA). I ​​don't think HUAWEI will provide good support for .NET.
